kserve/base/params.env

```diff
@@ -0,0 +1,8 @@
+kserve-controller=quay.io/opendatahub/kserve-controller:v0.10.2
```

Comment:
@israel-hdez do we need to update this to `latest` or `v0.11-latest` to avoid the anyuid issue?

Reply:
Manifests are for 0.10. So, we could use `v0.10-latest`, which should also be patched. Doesn't it work?

Or, do we want to move to 0.11? In upstream 0.11 is still unreleased. So, I'm not sure...

Reply:
I am ok any version that includes the changes regarding anyuid.